src/test/java/org/jenkinsci/plugins/docker/workflow/DockerUtilsTest.java

```diff
@@ -23,14 +23,17 @@
  */
 package org.jenkinsci.plugins.docker.workflow;
 
+import hudson.EnvVars;
 import hudson.FilePath;
+import hudson.slaves.EnvironmentVariablesNodeProperty;
 import org.hamcrest.collection.IsCollectionWithSize;
```

Comment:
Imports to facilitate changes to test
```suggestion
 import org.hamcrest.collection.IsMapWithSize;
```

Reply:
It seems we might be limited to hamcrest 1.3? MapWithSize seems not part of hamcrest 1.3:
http://hamcrest.org/JavaHamcrest/javadoc/1.3/

Is it feasible to use a different/newer hamcrest here?